Pod/INSImageView.swift

```diff
@@ -257,7 +296,7 @@ public class INSImageView: UIImageView {
     
     public override var image: UIImage? {
         get {
-            return nil
+            return imageView.image
```

Comment:
I understand getting the image is necessary so I've already previously implemented an originalImage property to do so.